translations/ui-users/en.json

```diff
@@ -93,6 +94,20 @@
   "settings.commentRequired": "Comment required",
   "settings.transferAccounts": "Transfer accounts",
   "settings.customFields": "Custom fields",
+  "settings.conditions": "Conditions",
+  "settings.feefine.balance": "Maximum outstanding fee/fine balance",
+  "settings.items.charged": "Maximum number of items charged out",
+  "settings.lost.items": "Maximum number of lost items",
+  "settings.overdue.items": "Maximum number of overdue items",
+  "settings.overdue.recalls": "Maximum number of overdue recalls",
+  "settings.overdue.recallMax": "Recall overdue by maximum number of days",
```

Comment:
None of these values are actually used. Rather, they correspond to records in the `patron-block-conditions` endpoint which makes me think that title field should be immutable, with values like `BLOCK_CONDITION_MAX_BALANCE` that other backend-code can key off when implementing such functionality. As things currently stand, I could PUT to that endpoint and change the name and break the system. Eeek! 

I realize that's way outside the scope of this PR, and further that this UI does _not_ give you the opportunity to create such a request, but the fact that it is even possible is troubling to me.